src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/Rules/RazorGenerateWithTargetPath.cs

```diff
@@ -10,58 +10,58 @@
 
```

Comment:
Why did we check in all these generated files in the first place - is it for CI/functional tests?

Also, I'd be hesitant to check in the changes to this file, as it'll likely revert on the next regeneration and show up on our git diffs.